jvb/src/main/kotlin/org/jitsi/videobridge/EndpointK.kt

```diff
@@ -362,6 +363,20 @@ class EndpointK @JvmOverloads constructor(
         }
     }
 
+    override fun getDebugState(): JSONObject {
+        return super.getDebugState().apply {
+            // debugState.put("sctpManager", sctpManager.getDebugState());
```

Comment:
Either convert the commented-out code to Kotlin, or remove it.

Reply:
Looks like `SctpManager` doesn't even have `getDebugState`, so removed it.